packages/block-library/src/group/variations.js

```diff
@@ -42,7 +42,10 @@ const variations = [
 			blockAttributes.layout?.orientation === 'vertical',
 		icon: stack,
 	},
-	{
+];
+
+if ( window.__experimentalEnableGroupGridVariation ) {
```

Comment:
I have a vague memory of folks mentioning that it's good to add optional chaining when attempting access the window object from the root of a JS file in the Gutenberg repo, but I can't _quite_ remember why. Possibly to do with mobile apps? At any rate, I see the `__experimentalEnableZoomedOutView` uses it here: https://github.com/WordPress/gutenberg/blob/e43d6500c2768ff7906ffc3d23dd7272e0baf765/packages/edit-site/src/components/header-edit-mode/index.js#L132

Should we do that here, too?

```suggestion
if ( window?.__experimentalEnableGroupGridVariation ) {
```